projects/slp.blue.json

```
@@ -25,33 +25,40 @@
     "main.png"
   ],
   "points": [
+    {
+      "icon": "drive_folder_upload.svg",
+      "title_cs": "Hladký přechod z původního SLP®",
+      "title_en": "",
+      "description_cs": "Pokud jste uživatelem programu SLP® od firmy SEKK®, do slp.blue automaticky převedete data včetně číselníků a stávající dokumentace. slp.blue nabízí přehledný a rychlý způsob kontroly převedených dokumentů.", 
```

můžete automaticky převést všechna data
```suggestion
      "description_cs": "Pokud jste uživatelem programu SLP® od firmy SEKK®, do slp.blue můžete automaticky převést všechna data včetně číselníků a stávající dokumentace. slp.blue nabízí přehledný a rychlý způsob kontroly převedených dokumentů.", 
```